sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -444,3 +448,123 @@ def display_data(self):
 
   def writer(self):
     raise NotImplementedError
+
+
+class MultipleReadFromPubSub(PTransform):
+  """A ``PTransform`` that expands ``ReadFromPubSub`` to read from multiple
+  subscriptions and/or topics."""
+  def __init__(
+      self,
+      source_list,  # type: List[str]
+      with_context=False,  # type: bool
```

Comment:
It seems like constructing `ReadFromPubSub` doesn't ask for `with_context`. Why do we add a new one here?